src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Random;
+
+@RestController
+@RequestMapping("/users")
 public class UserController {
+    private long key;
+
+    private UserMapper userMapper;
+
+    @Autowired
+    public UserController(UserMapper userMapper) {
+        this.userMapper = userMapper;
+    }
+
+    @PostMapping("createUser")
+    public User createUser(@RequestBody UserDto userDto) {
```

Comment:
1. To jest celowo, żeby przyjmować UserDto w requeście i zwracać User ?
2. Jeśli chodzi o standard to raczej się robi addUser a nie createUser. 

Może nie rozumiem zamysłu :) Jeśli to tak ma być to uprzejmie proszę o wyjaśnienie...

Reply:
Dalej podtrzymuję, że powinno być "addUser"